Unix/installbuilder/datafiles/Linux.data

```diff
@@ -189,6 +190,40 @@ ConfigureOmiService() {
     ${{SERVICE_CTL}} start
 }
 
+ConfigureLogRotate()
+{
+    echo "Configure log rotate for OMI"
+    # create the logrotate file if it doesn't exist
+    if [ ! -f /etc/logrotate.d/omi ]; then
+        cat /etc/opt/omi/conf/omilogrotate.conf > /etc/logrotate.d/omi
+    fi
+}
+
+ConfigureCronForLogRotate()
+{
+    echo "Checking if cron is installed..."
+    # warn user that he need to install cron if cron doesn't install
+    crontab -l >/dev/null 2>&1
+    if [ $? -ne 0 ]; then
+        echo "WARNING: LogRotate can't be enabled, please install cron at first!"
+        return
+    fi
+
+    echo "Checking if cron/crond service is started..."
+    # warn user that he need to start cron/crond service if cron doesn't start
+    cronid=$(pidof cron > /dev/null 2>&1)
+    crondid=$(pidof crond > /dev/null 2>&1)
+    if [ ! -z "$cronid" -a ! -z "$crondid" ]; then
+        echo "WARNING: LogRotate can be enabled, but please start cron/crond service!"
+    fi
+
```

Comment:
If "crontab -l" works, shouldn't cron service be running already?

Reply:
`crontab -l` works fine even cron service is stopped, thanks.

Reply:
Then the question is why even check for crontab -l?  According on online doc, crontab -l lists content of /var/spool/cron/crontabs directory.  You aren't using crontab command to add/remove cron entries, so it doesn't matter?

Reply:
you can check here: https://github.com/Microsoft/OMS-Agent-for-Linux/blob/aa3973a65343eb4b4e557b97766654765a34cc0f/docs/Troubleshooting.md#on-ubuntudebian , it works on all Linux, thanks.

Reply:
Cron is driven by a crontab (cron table) file, a configuration file that specifies shell commands to run periodically on a given schedule. But wiki.dreamhost for crontab mentiones : The crontab command, found in Unix and Unix-like operating systems, is used to schedule commands to be executed periodically.
If user don't have crontab, then the feature won't work, thanks.

Reply:
yes, you are right. so we should check `cron` or `crond` program instead of `crontab -l`? or just remove `crontab -l` checking logic?

BTW, our `config_keytab_update.sh` didn't check `crontab -l` program exist or not: https://github.com/Microsoft/omi/blob/a0aa5e42fa4e632aa7d96afe07b7f8ff3b55ec53/Unix/scripts/config_keytab_update.sh#L15, so we assume it should exist on all Linux otherwise we should get error when installation.

Reply:
Fixed: check `cron` or `crond` program instead of `crontab -l`, thanks.

Reply:
crontab is a basic command of Linux, and I'm not aware of any reason why it would not be in one's $PATH.  Though it doesn't hurt, it's also probably unnecessary.